src/app.module.ts

```diff
@@ -49,6 +49,7 @@ import { TransactionsViewControllerModule } from '@/routes/transactions/transact
 import { DelegatesV2Module } from '@/routes/delegates/v2/delegates.v2.module';
 import { AccountsModule } from '@/routes/accounts/accounts.module';
 import { NotificationsModuleV2 } from '@/routes/notifications/v2/notifications.module';
+import { ScheduleModule } from '@nestjs/schedule';
```

Comment:
I am not opinionated regarding this, but this will be the first datasource we aren't wrapping. What do you think re. this?

Reply:
I'd be in favor of not wrapping it, it would be a bit overkill to wrap the annotations part and I don't see a lot of benefit in doing it since this is a NestJS official package. And it seems doubtful to me we replace it, unless we shift the approach completely to something else, like queues, in which case I'd say the API would be different. Wdyt?

Reply:
Sounds good to me.

Unrelated but I think we should reconsider this with the config. module as well.